src/redux/actions/todoSagaActions/addItemSagaAction.ts

```diff
@@ -0,0 +1,20 @@
+import { TodoSagaActions } from './index';
+import { TodoItemType } from '../../../models';
+import { SignInPayload } from '../authActions/successSignInAction';
+
+export type AddItemSagaAction = {
+  type: TodoSagaActions.ADD_ITEM_SAGA;
+  payload: AddItemSagaActionPayload;
+};
+
+export type AddItemSagaActionPayload = {
+  newItem: TodoItemType;
+  userToken: SignInPayload['userToken'];
```

Comment:
the same, import type from reducer, not from payload 
check all your types!

Reply:
will be fixed